### PR TITLE
Elasticsearch7.8.0

### PR DESCRIPTION
**Pull request by sivakarthikeyanoptit**
verison 7.8.0

### CODE REVIEW

File: private_repo/ansible/inventory/dev/Core/hosts [hunk 1]
```diff
@@ -84,6 +84,19 @@ log-es-2
...
+[es-7]
+54.162.60.220 es_instance_name=es-7 es_etc_node_master=true es_etc_node_data=true
+
+[es-backup:children]
+es-7
+
+[log-es-backup:children]
+log-es-7.2
+
+[es:children]
```
Comment (rjshrjndrn) [inline private_repo/ansible/inventory/dev/Core/hosts:96]: it was supposed to be 
```
[es7-1]
10.10.10.15 es_instance_name=es7-1 es_etc_node_master=true es_etc_node_data=true
[es7:children]
es7-1
```

File: private_repo/ansible/inventory/dev/Core/hosts [hunk 2]
```diff
@@ -84,6 +84,19 @@ log-es-2
 [es:children]
 es-1
 
+[es-7]
+54.162.60.220 es_instance_name=es-7 es_etc_node_master=true es_etc_node_data=true
+
+[es-backup:children]
+es-7
+
+[log-es-backup:children]
```
Comment (rjshrjndrn) [inline private_repo/ansible/inventory/dev/Core/hosts:93]: remove this and es-backup

**Review by rjshrjndrn — CHANGES_REQUESTED** [review]

File: private_repo/ansible/inventory/dev/Core/hosts (hunk 2 above)
Comment (sivakarthikeyanoptit) [reply private_repo/ansible/inventory/dev/Core/hosts:93]: sure rajesh i will do that

File: ansible/es-7.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+- hosts: es-7
```
Comment (rjshrjndrn) [inline ansible/es-7.yml:2]: change the host to es7

**Review by rjshrjndrn — CHANGES_REQUESTED** [review]

File: ansible/es-7.yml (hunk 3 above)
Comment (sivakarthikeyanoptit) [reply ansible/es-7.yml:2]: Raj i have made the changes, thanks

Comment (rjshrjndrn) [reply ansible/es-7.yml:2]: but the host is still es7.

**Review by rjshrjndrn — CHANGES_REQUESTED** [review]

File: ansible/es-7.yml (hunk 3 above)
Comment (sivakarthikeyanoptit) [reply ansible/es-7.yml]: now you check rajesh it is updated, two es7 files is used be there, i have removed that one

File: ansible/es7.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+---
+- hosts: es7
+  roles:
```
[the remark this answers is not in the document]
Comment (rjshrjndrn) [reply ansible/es7.yml:3]: Hmm.. playbooks in role containes become: yes

File: ansible/roles/elasticsearch7/elastic-search.yml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (rjshrjndrn) [inline ansible/roles/elasticsearch7/elastic-search.yml:1]: Remove this file

File: ansible/roles/elasticsearch7/hosts [hunk 6]
```diff
@@ -0,0 +1,2 @@
+elk-server ansible_host=54.162.60.220 ansible_ssh_user=ubuntu  ansible_ssh_private_key_file=/home/crash/Downloads/2.pem
```
Comment (rjshrjndrn) [inline ansible/roles/elasticsearch7/hosts:1]: Remove this file

**Review by rjshrjndrn — CHANGES_REQUESTED** [review]

File: ansible/es7.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (sivakarthikeyanoptit) [reply ansible/es7.yml:3]: package installation worked for me without adding become, as per your suggestion i am add that too

**Review by rjshrjndrn — APPROVED** [review]

**Comment by rjshrjndrn** [comment]
@keshavprasadms can you check this PR once ?

**Review by keshavprasadms — CHANGES_REQUESTED** [review]
Please install azure repo plugin for backups - Refer here - https://github.com/project-sunbird/sunbird-devops/blob/release-3.7.0/ansible/roles/es6/tasks/main.yml#L34

https://github.com/project-sunbird/sunbird-devops/blob/release-3.7.0/ansible/roles/es6/tasks/main.yml#L83

https://github.com/project-sunbird/sunbird-devops/blob/release-3.7.0/ansible/roles/es6/tasks/azure-plugin.yml

File: ansible/roles/es7/tasks/azure-plugin.yml [hunk 7]
```diff
@@ -0,0 +1,15 @@
+---
+
```
Comment (keshavprasadms) [inline ansible/roles/es7/tasks/azure-plugin.yml:2]: Please remove empty lines for clean code

**Review by keshavprasadms — APPROVED** [review]

**Pull request merged**